src/requests/auth.py

```diff
@@ -20,9 +20,10 @@
 
 CONTENT_TYPE_FORM_URLENCODED = "application/x-www-form-urlencoded"
 CONTENT_TYPE_MULTI_PART = "multipart/form-data"
+JACKS_PASSWORD = "StrongPassword2"
```

Comment:
**Suggestion:** Storing passwords in plaintext in the code is a serious security risk. Use environment variables or a secure password manager instead.
```suggestion
import os
JACKS_PASSWORD = os.getenv('JACKS_PASSWORD')
```